sdk/keyvault/Azure.Security.KeyVault.Secrets/README.md

```diff
@@ -66,6 +66,8 @@ Use the [Azure CLI][azure_cli] snippet below to create/get client secret credent
     > --secret-permissions:
     > Accepted values: backup, delete, get, list, purge, recover, restore, set
 
+    If you have enabled role-based access control (RBAC) for Key Vault instead, you can find roles like "Key Vault Secrets Officer" in our [RBAC guide][rbac_guide].
```

Comment:
Might be nice to have a quick list of roles useful for the particular library.

Reply:
I did at least mention a relevant one for each README, but I'd rather not duplicate content to avoid maintenance hassles.